core/server/utils/index.js

```diff
@@ -21,6 +21,7 @@ utils = {
     /**
      * Timespans in seconds and milliseconds for better readability
      */
+    /* eslint-disable key-spacing
     ONE_HOUR_S:          3600,
```

Comment:
This is out-commented, maybe that's why 😬

Reply:
OMG YOU WIN 🍪